owncloudApp/src/main/java/com/owncloud/android/ui/activity/FolderPickerActivity.kt

```diff
@@ -155,6 +162,14 @@ open class FolderPickerActivity : FileActivity(),
 
     override fun onCreateOptionsMenu(menu: Menu?): Boolean {
         menuInflater.inflate(R.menu.main_menu, menu)
+        menuItem = menu!!.findItem(R.id.action_search)
+        searchView = menuItem.getActionView() as SearchView
+        searchView.maxWidth = Int.MAX_VALUE
+        searchView.queryHint = resources.getString(R.string.actionbar_search)
+        if (isInSpacesScreen) {
+            menuItem.setVisible(false)
+        }
+        menu!!.removeItem(menu!!.findItem(R.id.action_share_current_folder).itemId)
```

Comment:
We should avoid `!!` as much as we can, since it could lead to `NullPointerException`s. We can just surround all these lines by a `menu?.{ ... }`, which will only run if menu is not `null`

Reply:
Ok. It's done ✅